itools-packager/src/main/java/com/powsybl/itools/ItoolsPackagerMojo.java

```diff
@@ -194,9 +288,16 @@ public void execute() {
             Files.createDirectories(libDir);
             copyFiles(copyToLib, libDir);
 
-            getLog().info("Zip package");
```

Comment:
Why do you remove this log?

Reply:
In this evolution, I moved this log to createPackage.